### ISSUE TITLE
Deprecate the use of `entities.System` in favor of `entities.Host`

### ISSUE THREAD

**Issue by omaciel** [SatelliteQE/nailgun#270]
As per this [issue](http://projects.theforeman.org/issues/13904) it seems that we should be using `entities.Host` whenever we used to use `entities.System`. This needs to be fully investigated.

**Comment by omaciel** [comment]
We should also update `Robottelo` existing tests to make sure that it uses `entities.Hosts`.

**Comment by JacobCallahan** [comment]
specifically the targeted api paths are below. the systems path still exists, but will hopefully be removed in a later version.

```
"host_subscriptions": {
      "List a host's subscriptions": "/api/hosts/:host_id/subscriptions",
      "Trigger an auto-attach of subscriptions": "/api/hosts/:host_id/subscriptions/auto_attach",
      "List subscription events for the host": "/api/hosts/:host_id/subscriptions/events",
      "Add a subscription to a host": "/api/hosts/:host_id/subscriptions/add_subscriptions",
      "Set content overrides for the host": "/api/hosts/:host_id/subscriptions/content_override",
      "Get content and overrides for the host": "/api/hosts/:host_id/subscriptions/product_content"
    },
```

**Comment by oshtaier** [comment]
@JacobCallahan 
It doesn't seem that API is still exist
HTTPError: 500 Server Error: Internal Server Error for url: https://server/katello/api/v2/systems

and basically https://github.com/Katello/katello/pull/5831/files shows that System API is going to be removed completely

**Comment by JacobCallahan** [comment]
@oshtaier oh nice it looks like they finally removed the path. Unfortunately, it is still listed in the GET to /api/v2/

**Comment by elyezer** [comment]
@JacobCallahan don't believe on that docs, since it says docker registries are there and they are not. It raise a 404 when trying to access any of the registry paths.

**Comment by JacobCallahan** [comment]
@elyezer haha ok :). I hope there is already a bug open for that.

**Comment by elyezer** [comment]
@JacobCallahan yes, there is.

**Comment by abalakh** [comment]
robottelo part is done in SatelliteQE/robottelo#3384 and SatelliteQE/robottelo#3391.
Regarding nailgun - 99% is done in #276 and #277 , 1 more small fix is required though:
* Un-ignore host attribute for HostCollection entity as soon as BZ1325989 is fixed (simply drop `HostCollection.read()`)

So... Waiting for BZ1325989 so we can finally close this issue :)

**Comment by abalakh** [comment]
The last part of this issue was resolved in #314, now we can finally close it
